packages/ember-views/lib/system/render_buffer.js

```diff
@@ -137,16 +137,21 @@ var canSetNameOnInputs = (function() {
   @namespace Ember
   @param {String} tagName tag name (such as 'div' or 'p') used for the buffer
 */
-export default function renderBuffer(tagName, contextualElement) {
-  return new _RenderBuffer(tagName, contextualElement); // jshint ignore:line
+export default function renderBuffer(tagName, contextualElement, _helper) {
+  return new _RenderBuffer(tagName, contextualElement, _helper); // jshint ignore:line
 }
 
-function _RenderBuffer(tagName, contextualElement) {
+function _RenderBuffer(tagName, contextualElement, _helper) {
   this.tagName = tagName;
   this._outerContextualElement = contextualElement;
   this.buffer = null;
   this.childViews = [];
-  this.dom = environment.hasDOM ? new DOMHelper() : null;
+
+  if (_helper) {
+    this.dom = _helper;
+  } else {
+    this.dom = environment.hasDOM ? new DOMHelper() : null;
```

Comment:
question: Is it possible to always externalize this branch? Basically making helper a required argument of `_RenderBuffer`